homeassistant/components/nissan_leaf/device_tracker.py

```diff
@@ -0,0 +1,46 @@
+"""Support for tracking a Nissan Leaf."""
+import logging
+
+from homeassistant.components.nissan_leaf import (
+    DATA_LEAF, DATA_LOCATION, SIGNAL_UPDATE_LEAF)
+from homeassistant.helpers.dispatcher import dispatcher_connect
+from homeassistant.util import slugify
+
+_LOGGER = logging.getLogger(__name__)
+
+DEPENDENCIES = ['nissan_leaf']
+
+ICON_CAR = "mdi:car"
+
+
+def setup_scanner(hass, config, see, discovery_info=None):
+    """Set up the Nissan Leaf tracker."""
+    _LOGGER.debug("Setting up Scanner (device_tracker) for Nissan Leaf, "
```

Comment:
Remove this.

Reply:
Fixed with commit https://github.com/filcole/home-assistant/commit/119b2a3b20db6ea4cf05fda29921930c5e27bc86